### PR TITLE
Pumpkin U-Boot configuration and misc

### PR DESCRIPTION
**Pull request by CatGarab**
**PR Overview**:

- U-Boot configuration file for Pumpkin board.
- Companion PR to kubostech/uboot#15
- Created board-specific overlay folders. fstab and fw_env.config are no longer portable.
- kubos-kernel.its, which defines how the kernel.itb file gets created, is not portable because of the different RAM load addresses
- Added post-image.sh and genimage.cfg which generates an SD card image a bit more nicely than our current format-image.sh script.

**Documentation**:

Added to-do items to [the Pumpkin doc card](https://trello.com/c/2KD8GDBx/15-kubos-linux-docs-updates-for-pumpkin)

**Integration Tests**:

- No integration tests. Still need a Pumpkin target to exist in the SDK.

### CODE REVIEW

**Comment by CatGarab** [comment]
[Pumpkin uboot](https://trello.com/c/vc77lz04/8-pumpkin-uboot)

**Comment by CatGarab** [comment]
[Pumpkin Boot from eMMC](https://trello.com/c/ry6KZw6F/32-pumpkin-boot-from-emmc)

File: board/kubos/pumpkin-mbm2/kubos-kernel.its [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+/dts-v1/;
+
+/ {
+	description = "KubOS Linux Kernel";
+	#address-cells = <1>;
+
+	images {
+		kernel@1 {
+			data = /incbin/("/home/catherine/glib/buildroot-2016.11/output-pumpkin/images/zImage");
```
Comment (plauche) [inline board/kubos/pumpkin-mbm2/kubos-kernel.its]: This path looks like its hard coded for your VM

Comment (CatGarab) [inline board/kubos/pumpkin-mbm2/kubos-kernel.its]: It is. I fixed that in #50, which builds on top of this.

**Pull request merged**